Description:
 Increase PJ_ICE_MAX_CAND to 256. Upstream's default, 16, is
 not enough to handle cases with numerous local/remote candidates
 like the case when multiple hosts in IPv6 and IPv4 exists.
 Tests show that 40 entries are needed with 5 local host IPs.
 .
 Increase PJ_ICE_MAX_CHECKS, which would be way too low for the
 number of candidates.
 .
 Increase PJ_ICE_MAX_STUN to 6:
  - One active, one passive for IPv4
  - One active, one passive for IPv6
  - One active, one passive set by the user
 .
 Increase PJ_ICE_COMP_BITS so that we can have more components:
  - 2 Audio components
  - 2 Video components
Author: Guillaume Roguez <guillaume.roguez@savoirfairelinux.com>

 pjnath/include/pjnath/config.h | 8 ++++----
 1 file changed, 4 insertions(+), 4 deletions(-)

diff --git a/pjnath/include/pjnath/config.h b/pjnath/include/pjnath/config.h
index 5c2f04bd..d8b546c8 100644
--- a/pjnath/include/pjnath/config.h
+++ b/pjnath/include/pjnath/config.h
@@ -232,7 +232,7 @@
  * Default: 16
  */
 #ifndef PJ_ICE_MAX_CAND
-#   define PJ_ICE_MAX_CAND			    16
+#   define PJ_ICE_MAX_CAND			    256
 #endif
 
 
@@ -253,7 +253,7 @@
  * Default: 4 (one active and one passive for IPv4 and IPv6)
  */
 #ifndef PJ_ICE_MAX_STUN
-#   define PJ_ICE_MAX_STUN			    4
+#   define PJ_ICE_MAX_STUN			    6
 #endif
 
 
@@ -273,7 +273,7 @@
  * the maximum number of components (PJ_ICE_MAX_COMP) value.
  */
 #ifndef PJ_ICE_COMP_BITS
-#   define PJ_ICE_COMP_BITS			    1
+#   define PJ_ICE_COMP_BITS			    2
 #endif
 
 
@@ -324,7 +324,7 @@
  * Default: 32
  */
 #ifndef PJ_ICE_MAX_CHECKS
-#   define PJ_ICE_MAX_CHECKS			    32
+#   define PJ_ICE_MAX_CHECKS			    1024
 #endif
 